requirements.txt

```diff
@@ -32,7 +32,7 @@ flask-session==0.3.1
 flask-sqlalchemy==2.3.2
 flask-swagger==0.2.13
 flask-testing==0.7.1
-flask-user==0.6.21 # pyup: <0.7 # pin until 1.0 is ready for prod
+git+https://github.com/uwcirg/Flask-User.git@v0.6.21#egg=flask-user # pyup: <0.7 # pin until 1.0 is ready for prod
```

Comment:
*Almost* :)

Paul helped me repro this, but you'll need to increment the version in your fork if you want it to be installed over (ie replace) the vanilla un-forked flask-user. If you create a fresh virtual environment this will work, but won't work for a venv that already has flask-user installed since pip thinks it's already installed at the same version.

You'll want to pick a version that's higher than `v0.6.21`, but that won't conflict with the author's next release. Judging from their previous releases, I would suggest `v0.6.21.1`.

Also, I'm not sure how picky pip is when comparing versions, or installing from git sources using a git tag. If you have any issues, I would suggest your [`git tag`](https://github.com/uwcirg/Flask-User/tags) **exactly** match the version in `requirements.txt`, ie don't include `_with_password_failed_signal` :)

Reply:
Ah nice! I've only figured out what works for my narrow use-case, will have to try that in the future :)